crates/bevy_ui/src/lib.rs

```diff
@@ -22,44 +23,55 @@ pub mod picking_backend;
 
 use bevy_derive::{Deref, DerefMut};
 use bevy_reflect::{std_traits::ReflectDefault, Reflect};
+#[cfg(all(feature = "bevy_text", feature = "bevy_render"))]
 mod accessibility;
```

Comment:
`accessibility` shouldn't need text. It's using the text component to store strings, but they are meant to be read using a screenreader I think. I'm not sure the module works or makes sense though, so it doesn't matter I guess?

Reply:
I agree. The module uses Label extensively, which I think uses Text. I'll double check that.